src/components/ClaimForm.js

```diff
@@ -221,6 +226,13 @@ class ClaimForm extends Component {
     if (!d[type]) return false;
     if (d.qtyProvided === null || d.qtyProvided === undefined || d.qtyProvided === "") return false;
     if (d.priceAsked === null || d.priceAsked === undefined || d.priceAsked === "") return false;
+    if (
+      this.explanationRequiredIfQuantityAboveThreshold &&
+      !d.explanation &&
```

Comment:
This has to be changed, it will block both services and items. Only services should be blocked.